### PR TITLE
control plane release 2022-11-17

### PR DESCRIPTION
**Pull request by antonyc**
Preview:
https://websitemain62807-antonycpatch159718.gatsbyjs.io/docs/release-notes/

### CODE REVIEW

File: content/release-notes/2022-11-17.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+label: 'Console'
+---
+
+### What's new
+
+- UI: Postgres 15 is now the default version when creating a new project.
+- UI: Branches can be renamed in the single branch view screen.
+- UI: Added timestamp and lsn to the branch view **TODO**: find out what it is https://github.com/neondatabase/cloud/pull/2919/commits/d05c714845e4550fa7f460755e9bbb8c778e00fe
```
Comment (antonyc) [inline content/release-notes/2022-11-17.md:9]: @seymourisdead can you please clarify "add timestamp and lsn to the branch view" for [this commit](https://github.com/neondatabase/cloud/pull/2919/commits/d05c714845e4550fa7f460755e9bbb8c778e00fe)?

File: content/release-notes/2022-11-17.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+- UI: Added branch icon **TODO** https://github.com/neondatabase/cloud/pull/2777 where was it added to?
```
Comment (antonyc) [inline content/release-notes/2022-11-17.md:10]: @duskpoet can you please clarify where was this icon added in https://github.com/neondatabase/cloud/pull/2777 ?

**Review by danieltprice — APPROVED** [review]
If you're good with the changes, @antonyc, let's merge this.

**Review by danieltprice — APPROVED** [review]
lgtm!

**Pull request merged**